Explorer++/TestExplorer++/ShellHelperTest.cpp

```diff
@@ -64,6 +65,15 @@ TEST_F(TransformPathTest, EnvironmentVariablesExpansion)
 	set = SetEnvironmentVariable(L"ShellFolderPath", L"shell:public");
 	ASSERT_TRUE(set);
 	PerformTest(L"%ShellFolderPath%", currentDirectory, L"shell:public");
+
+	// Nested environment variable expansion.

```

Comment:
Since `GetExpandedEnvironmentVariable` appears in `Helper.cpp`, this test should be moved to `HelperTest.cpp`.